tests/unit/test_run_logprep.py

```diff
@@ -525,3 +527,25 @@ def test_pseudonymize_depseudonymize_with_mode(self, mode, tmp_path):
         )
         assert result.exit_code == 0
         assert result.output.strip() == "string"
+
+
+class TestYamlLoaderTags:
+    def test_yaml_loader_include_tag_initialized(self):
+        yaml = YAML(pure=True, typ="safe")
+        target_directory = tempfile.mkdtemp()
+        path_to_file_to_include = self._write_to_yaml_file("this: was included", target_directory)
```

Comment:
consider to use the `tmp_path` fixture for this. 

have a look at: https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html